src/App.jsx

```diff
@@ -14,45 +19,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState('');
+
+  const sortByAlphabet = () => {
```

Comment:
you shouldn't sort, reverse array in these functions
just set correct `setSortField` or `setReversed` state values

create a function that accepts these values and sorts an array of goods in the way you need and returns the sorted array

use this function in the component to get a sorted array of goods. It will be recalculated on every state change (state change leads to re-render)